pkg/upgrade/upgrades/create_auto_config_runner_job.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package upgrades
+
+import (
+	"context"
+
+	"github.com/cockroachdb/cockroach/pkg/clusterversion"
+	"github.com/cockroachdb/cockroach/pkg/jobs"
+	"github.com/cockroachdb/cockroach/pkg/jobs/jobspb"
+	_ "github.com/cockroachdb/cockroach/pkg/jobs/metricspoller" // Ensure job implementation is linked.
+	"github.com/cockroachdb/cockroach/pkg/security/username"
+	"github.com/cockroachdb/cockroach/pkg/sql/isql"
+	"github.com/cockroachdb/cockroach/pkg/sql/sessiondata"
+	"github.com/cockroachdb/cockroach/pkg/upgrade"
+)
+
+func createAutoConfigRunnerJob(
+	ctx context.Context, _ clusterversion.ClusterVersion, d upgrade.TenantDeps,
+) error {
+	if d.TestingKnobs != nil && d.TestingKnobs.SkipAutoConfigRunnerJobBootstrap {
+		return nil
+	}
+	return d.DB.Txn(ctx, func(ctx context.Context, txn isql.Txn) error {
+		row, err := d.DB.Executor().QueryRowEx(
+			ctx,
+			"check for existing auto config runner job",
+			nil,
+			sessiondata.InternalExecutorOverride{User: username.RootUserName()},
```

Comment:
Two things here:
1) Why root and not node? (we are so unprincipled about it)
2) Want to just use the var in that package which is this value?